### PR TITLE
Add support for multiple Streamlit panels.

### LINKED ISSUE #3: Opening a second report breaks streamlit-atom
## Summary

Opening a second report on top of a first creates the following error:
```
Error: The workspace can only contain one instance of item [object Object]
```

## Steps to reproduce.

1. Open a report in `streamlit-atom` (`ctrl-opt-O`)
2. Open a second source file and open _it_ (`ctrl-opt-O`)

## What I Expect To Happen

_One_ of the following:

1. A new tab opens with the new report. Tabs are named after the report names, OR
2. The current existing tab reloads the new report.

I _think_ (1) is preferable, but am not sure.

### PR DESCRIPTION
**Pull request by tvst**
This does 3 things:
1. Fixes #3 , so you can now view multiple reports at the same time.
2. Adds a real README.md
3. Replaces "host" and "port" settings for a single "URL".

### CODE REVIEW

File: lib/streamlit-atom-view.js [hunk 1]
```diff
@@ -43,10 +53,21 @@ export default class StreamlitAtomView {
...
+    this.navigateTo(url, scriptName);
+    return atom.workspace.open(this);
+  }
+
+  // The following are the interface for the item in the pane.
+  // See API: https://github.com/atom/atom/blob/master/src/workspace.js
+
   getTitle() {
-    return 'Streamlit';
+    return `${this.scriptName} 🗲 Streamlit`;
```
Comment (aaj-st) [inline lib/streamlit-atom-view.js]: i dont know what the box character is?  unicode?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,82 @@
...
+
+
+## Settings
+
+Streamlit Atom has the following settings, which you can access through Atom's
+usual Settings View (`ctrl-,`/`cmd-,`):
+
+* **Streamlit proxy URL**: the URL of the Streamlit proxy, which is where the
+  Streamlit report is served from. When editing files locally the defaults
+  should _just work_ (i.e. `localhost:8501`).
```
Comment (aaj-st) [inline README.md]: Is this one config option now?  Wasnt it 2 options before, host and port vs just url?  Looking at the code it still shows two options.

Comment (tvst) [reply README.md]: You're right. See the latest commit that I added to this PR.

File: lib/streamlit-atom-view.js (hunk 1 above)
Comment (tvst) [reply lib/streamlit-atom-view.js]: Oh, that's annoying. It's a cool lightning bolt :)

I'll remove it, since it doesn't show on a Mac.

**Pull request merged**